release-notes/opensearch-ml-common.release-notes-2.1.0.0.md

```diff
@@ -0,0 +1,16 @@
+## Version 2.1.0.0 Release Notes
+
+Compatible with OpenSearch 2.1.0
+
+### Features
+
+* dispatch ML task to ML node first ([#346](https://github.com/opensearch-project/ml-commons/pull/346))
+
+### Enhancements
+
+* Use custom plugin to publish zips to maven ([#343](https://github.com/opensearch-project/ml-commons/pull/343))
```

Comment:
This change seems already released in https://github.com/opensearch-project/ml-commons/releases/tag/2.0.1.0

Reply:
ic, will remove it